src/js/redux/reducers/search/columnVisibilityReducer.js

```diff
@@ -0,0 +1,251 @@
+/**
+ * columnVisibilityReducer.js
+ * Created by Lizzie Salita 5/04/17
+ **/
+
+import { Record, OrderedSet } from 'immutable';
+
+import * as ColumnVisibilityFuncs from './filters/ColumnVisibilityFunctions';
+
+const initialState = {
```

Comment:
If we're going to have 60+ columns, let's move this into its own file. Also if you `export` the initialState, you can use it in your tests without having to copy it over there and maintain two versions